roles/sap_anydb_install_oracle/tasks/oracledb_install_pre.yml

```diff
@@ -0,0 +1,133 @@
+    - libaio
+    - libaio-devel
+    - libgcc
+    - libstdc++
+    - libstdc++-devel
+    - make
+    - psmisc
+    - rdma-core-devel
+    - smartmontools
+    - sysstat
+    - unzip
+    - xorg-x11-utils
+    - xorg-x11-xauth
+    - unixODBC
+    - unixODBC-devel
+    - libnsl
+    state: present
+  notify: __oracledb_prep_reboot_handler
+
+# Removed Packages from RHEL 8.x
+# https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/considerations_in_adopting_rhel_8/index#removed-packages_changes-to-packages
+#    - compat-libcap1
+#    - compat-libstdc++-33
+
+
+- name: Oracle DB - Create Linux User Groups
+  become_user: root
+  ansible.builtin.group:
+    name: "{{ item.group }}"
+    gid: "{{ item.gid }}"
+    state: present
+  loop:
+    - { group: oinstall, gid: '1001' }
+    - { group: dba, gid: '1002' }
+    - { group: oper, gid: '1003' }
+    - { group: oracle, gid: '1004' }
+
+- name: Oracle DB - Create Linux Users
+  become_user: root
+  ansible.builtin.user:
+    name: "{{ item.name }}"
+    uid: "{{ item.uid }}"
+    groups: "{{ item.groups }}"
+    shell: "{{ item.shell }}"
+    comment: "{{ item.comment }}"
+    append: yes
+    state: present
+  with_items:
+    - { name: 'oracle', uid: '1003', groups: 'oinstall,dba,oper', shell: '/bin/csh', comment: 'Oracle System Administrator' }
+
+- name: Oracle DB - Create directories, from root user
+  become_user: root
+  ansible.builtin.file:
+    state: directory
+    path: "{{ item.path }}"
+    owner: "{{ item.owner }}"
+    group: "{{ item.group }}"
+    mode: '0774'
+  loop: 
+    - { path: "{{ oracledb_base }}", owner: oracle, group: oinstall } #  Oracle Base directory
+    - { path: "{{ oracledb_base }}/dvd/19.0.0", owner: oracle, group: oinstall } #  Oracle home image file path
+    - { path: "{{ oracledb_base }}/{{ oracledb_sid }}/19.0.0", owner: oracle, group: oinstall } #  Oracle Home installation directory
+    - { path: "{{ oracledb_inventory_central }}", owner: oracle, group: oracle } #  Oracle Central Inventory
+    - { path: "{{ oracledb_base }}{{ oracledb_inventory_central }}", owner: oracle, group: oinstall } #  Oracle Inventory
+    - { path: "{{ oracledb_filesystem_storage }}", owner: oracle, group: oinstall }
+    - { path: /rman, owner: oracle, group: oinstall }
+    - { path: /archivelog, owner: oracle, group: oinstall }
+
+- name: Oracle DB - Copy installation media
+  ansible.builtin.copy:
+    remote_src: true
+    directory_mode: true
+    src: "{{ oracledb_installer_extract_path }}/LINUX_X86_64/"
+    dest: "{{ oracledb_base }}/dvd/19.0.0"
+    owner: oracle
+    group: oinstall
+    mode: '0774'
+
+# Requires change directory to Oracle RDBMS installation media directory
+# Must extract to run the precheck
+- name: Oracle DB - check installation prerequisites for Oracle DB
+  become: true
+  become_user: oracle
+  args:
+    executable: /bin/csh
+    chdir: "{{ oracledb_base }}/dvd/19.0.0/"
+  register: oracledb_precheck
```

Comment:
[style guide](https://sap-linuxlab.github.io/contribute/contribute_04_dev_guidelines/): rename this variable to __sap_anydb_install_oracle_precheck (for example). Similarly for the other 6 places in this role.